packages/datadog-plugin-mysql2/src/index.js

```diff
@@ -1,9 +1,43 @@
 'use strict'
 
-const MySQLPlugin = require('../../datadog-plugin-mysql/src')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
 
-class MySQL2Plugin extends MySQLPlugin {
+class MySQL2Plugin extends DatabasePlugin {
   static get name () { return 'mysql2' }
+  static get system () { return 'mysql' }
+
+  start ({ sqlStatement, sql, conf: dbConfig }) {
+    const service = getServiceName(this.config, dbConfig)
+
```

Comment:
It looks like everything here is re-implemented from the base class. Can you just call `super.start()`? (Fine if not.)

Reply:
@bengl I don't think I can, I changed the base class to be the Database class, before we had MySQL2 extend the MySQL class.

Reply:
If it can't just be inherited, maybe the repeated part could be extracted to another method that is called in both `start` methods?

Reply:
@rochdev we have fixed the issue with pass by reference and have reverted back to extending the MySQL plugin.